frontend/src/libs/components/order-list/order-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { type PlaceLatLng } from '~/libs/packages/map/libs/types/types.js';
+import { type OrderResponseDto } from '~/libs/types/types.js';
+
+import { OrderListCardBusiness } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  orders: OrderResponseDto[];
+  select: ({ startPoint, endPoint }: PlaceLatLng) => void;
```

Comment:
```suggestion
  onSelect: ({ startPoint, endPoint }: PlaceLatLng) => void;
```

Reply:
I definitely changed it. I have no idea why it hasn't changed. Maybe autoresolve conflicts. Redid.